module/VuFind/src/VuFind/Recommend/ConsortialVuFindDeferred.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * ConsortialVuFindDeferred Recommendations Module
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2020.
```

Comment:
Maybe update date to 2023 since this is new?

Reply:
This whole class was a mistaken commit -- I decided (for now) not to introduce a deferred version.